docs/explanations.rst

```diff
@@ -233,3 +264,186 @@ After that, is just a matter of running the file specifying which example to use
     ./examples.py 18
 
 We encourage you to adapt/improve/hack the examples in the file to play with different combinations of message types to learn and "feel" how the output would be in the different cases.
+
+
+Understanding which/how messages are shown/logged
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is how texts are exposed to the screen for the different situations according to the selected verbosity level by the user running the application.
+
+The last column of the table though is not about the screen: it indicates if the information will be present in the log created automatically by Craft CLI.
+
+.. list-table::
+   :header-rows: 1
+
+   * -
+     - QUIET
+     - BRIEF
+     - VERBOSE
+     - DEBUG
+     - TRACE
+     - also to logfile
+   * - ``.message(...)``
+     - --
+     - | stdout
+       | permanent
+       | plain
```

Comment:
Some of the "plain" strings have different styling but I think the github diff is confused with markup instead of showing verbatim content. I'm assuming this is correct in the source code.

Reply:
The table builds ok and consistent, yes. Thanks!